pkg/varnishcluster/controller/varnishcluster_grafana_dashboard_test.go

```diff
@@ -78,6 +80,12 @@ var _ = Describe("grafana dashboard", func() {
 				"foo":                       "bar",
 			}))
 
+			By("Dashboard title should default to <cluster name> varnish")
+			dashboardString := dashboardCM.Data[names.GrafanaDashboardFile(newVC.Name)]
+			var data map[string]interface{}
+			_ = json.Unmarshal([]byte(dashboardString), &data)
```

Comment:
I'd rather not ignore the error. In that case if someone accidentally breaks the JSON we'll have a clear unmarshal error compared to not having the "title" in the map assertion fail. Easier in debugging.
Same in the next test.